stammer.py

```diff
@@ -258,13 +275,17 @@ def main():
     parser.add_argument('output_path', type=Path, metavar='output_file', help='path to file that will be written to; should have an audio or video file extension (such as .wav, .mp3, .mp4, etc.)')
     args = parser.parse_args()
     
-    try:
-        TEMP_DIR.mkdir()
```

Comment:
I'm not sure why replacing this is necessary.

[`mkdir()`](https://docs.python.org/3/library/pathlib.html#pathlib.Path.mkdir) will raise `FileExistsError` if the the directory already exists, so you could just add an except clause like:
```py
    except FileExistsError:
        print("Temp directory was found when it should have been deleted. Deleting... Please try again.")
        shutil.rmtree(TEMP_DIR)  # This will completely delete TEMP_DIR
```
Furthermore, why do we want to stop auto-deleting `temp/` in this case? Prior to this change, it will delete the directory if it detects it exists.

Reply:
1. I don't like the verbose traceback output that `except` has generated in my testing. If I can make this pythonic way of things work with how I want *output* to be, then sure, I'll do it that way! TLDR;  I was short on time, and the code does the thing so eh :^)
2. I consider auto-deleting very "disk destroyer" behavior, especially because here it encourages two instances of the same program to fight for the same paths.